CHANGELOG.asciidoc

```diff
@@ -32,6 +32,7 @@ allows for spans representing asynchronous handling of requests for which the co
 
 [float]
 ===== Features
+* Set the service version when using the ECS reformatting of the application logs: {pull}2603[#2603]
```

Comment:
Do you think this is a breaking change for people using additional field for setting the service version?

Reply:
Good question, depends on what happens when we set both - would we serialize the same field twice? What would be the consequences? You can try to set two different versions and see what happens.

Either way, I think adding a bullet under a `Potentially breaking changes` section in the changelog can't harm.

Reply:
It'll depend on the JSON processor and it's configuration. I've recently enhanced the Elasticsearch [json processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/json-processor.html) to add a flag for `allow_duplicate_keys` for this reason. I don't remember how Beats handles duplicate keys. It might allow them but I'm not quite sure.

Reply:
So do you suggest anything other than adding a sentence to a `Potentially breaking changes` section? Such sentence would say that such additional key should be removed and either rely on automatic discovery or on agent config.

Reply:
SGTM